.github/workflows/release.yml

```diff
@@ -42,6 +42,8 @@ jobs:
           mv jsr.json.tmp jsr.json
       - run: yarn install
       - run: 'mkdir -p out && yarn pack --out out/%s-%v.tgz'
+      - name: Check Package Entrypoints
+        run: 'yarn attw out/*.tgz'
```

Comment:
This is the part I’m not sure about.

Should we have any smoke test before pushing the package, or is this not the right place?